clusterloader2/testing/load/config.yaml

```diff
@@ -78,6 +78,7 @@
 {{$defaultQps := DefaultParam .CL2_DEFAULT_QPS (IfThenElse (le .Nodes 500) 10 100)}}
 
 {{$ADDITIONAL_MEASUREMENT_MODULES := DefaultParam .CL2_ADDITIONAL_MEASURMENT_MODULES nil}}
+{{$ADDITIONAL_OBJECTS_MODULES := DefaultParam .CL2_ADDITIONAL_OBJECTS_MODULES nil}}
```

Comment:
Could we rename this to `ADDITIONAL_PHASES_MODULES` and `CL2_ADDITIONAL_PHASES_MODULES`?

This sounds more in line with the CL2's API where a single step can be either a list of measurements or phases.

Reply:
Done